lib/node_modules/@stdlib/assert/is-same-booleanarray/README.md

```diff
@@ -0,0 +1,122 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# isSameBooleanArray
+
+> Test if two arguments are both [BooleanArrays][@stdlib/array/bool] and have the [same values][@stdlib/assert/is-same-value].
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var isSameBooleanArray = require( '@stdlib/assert/is-same-booleanarray' );
+```
+
+#### isSameBooleanArray( v1, v2 )
+
+Tests if two arguments are both [BooleanArrays][@stdlib/array/bool] and have the [same values][@stdlib/assert/is-same-value].
+
+```javascript
+var BooleanArray = require( '@stdlib/array/bool' );
+
+var x = new BooleanArray( [ true, false ] );
+var y = new BooleanArray( [ true, false ] );
+var bool = isSameBooleanArray( x, y );
+// returns true
+
+bool = isSameBooleanArray( x, [ true, false ] );
+// returns false
+```
+
+</section>
+
+<!-- /.usage -->
+
+<section class="notes">
+
+## Notes
+
+-   In contrast to the strict equality operator `===`, the function distinguishes between `+0` and `-0` and treats `NaNs` as the [same value][@stdlib/assert/is-same-value].
+
+</section>
+
+<!-- /.notes -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var BooleanArray = require( '@stdlib/array/bool' );
+var isSameBooleanArray = require( '@stdlib/assert/is-same-booleanarray' );
+
+var x = new BooleanArray( [ true, false, false, true ] );
+var y = new BooleanArray( [ true, false, false, true ] );
+var out = isSameBooleanArray( x, y );
+// returns true
+
+x = new BooleanArray( [ true, false, false, true ] );
+y = new BooleanArray( [ true, true, false, false ] );
+out = isSameBooleanArray( x, y );
+// returns false
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for related `stdlib` packages. Do not manually edit this section, as it is automatically populated. -->
+
+<section class="related">
+
+* * *
+
+## See Also
+
+-   <span class="package-name">[`@stdlib/assert/is-booleanarray`][@stdlib/assert/is-booleanarray]</span><span class="delimiter">: </span><span class="description">test if a value is a BooleanArray.</span>
+-   <span class="package-name">[`@stdlib/assert/is-same-uint8array`][@stdlib/assert/is-same-uint8array]</span><span class="delimiter">: </span><span class="description">test if two arguments are both Uint8Arrays and have the same values.</span>
+-   <span class="package-name">[`@stdlib/assert/is-same-value`][@stdlib/assert/is-same-value]</span><span class="delimiter">: </span><span class="description">test if two arguments are the same value.</span>
```

Comment:
```suggestion
```

This section, as mentioned in the comment, is handled by tooling. It should not be manually populated for new packages.